scripts/circleci-build-erlang.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -x
+set -e
+if [[ ! -d ~/.kerl/$OTP_VERSION ]]; then
+    if [[ ! -d ~/.kerl ]]; then
+        mkdir ~/.kerl
+        curl -fsSLo ~/.kerl/kerl https://raw.githubusercontent.com/kerl/kerl/master/kerl
+        chmod +x ~/.kerl/kerl
+        ~/.kerl/kerl update releases
```

Comment:
Could you move `update releases` below `fi` (on below line)?
Otherwise this PR looks good.